Traceback (most recent call last):
  File "/Users/te/opt/anaconda3/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/Users/te/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/te/opt/anaconda3/lib/python3.7/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/te/opt/anaconda3/lib/python3.7/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/Users/te/opt/anaconda3/lib/python3.7/asyncio/base_events.py", line 583, in run_until_complete
    return future.result()
  File "/Users/te/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 554, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/Users/te/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/te/opt/anaconda3/lib/python3.7/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
if reverse_string('Hello') == 'olleH':
    print('‡∏ñ‡∏π‡∏Å‡∏ï‡πâ‡∏≠‡∏á')
else:
    print('‡πÑ‡∏°‡πà‡∏ñ‡∏π‡∏Å‡∏ï‡πâ‡∏≠‡∏á')
    
if reverse_string('Welcome to the new World!') == '!dlroW wen eht ot emocleW':
    print('‡∏ñ‡∏π‡∏Å‡∏ï‡πâ‡∏≠‡∏á')
else:
    print('‡πÑ‡∏°‡πà‡∏ñ‡∏π‡∏Å‡∏ï‡πâ‡∏≠‡∏á')
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m/var/folders/j7/mgs164kx5xx37ltn1q1f2_sw0000gn/T/ipykernel_27147/2816149918.py[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0;32mif[0m [0mreverse_string[0m[0;34m([0m[0;34m'Hello'[0m[0;34m)[0m [0;34m==[0m [0;34m'olleH'[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      2[0m     [0mprint[0m[0;34m([0m[0;34m'‡∏ñ‡∏π‡∏Å‡∏ï‡πâ‡∏≠‡∏á'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      3[0m [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m     [0mprint[0m[0;34m([0m[0;34m'‡πÑ‡∏°‡πà‡∏ñ‡∏π‡∏Å‡∏ï‡πâ‡∏≠‡∏á'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m [0;34m[0m[0m

[0;31mNameError[0m: name 'reverse_string' is not defined
NameError: name 'reverse_string' is not defined

